sql/updates/pre-0.1.0--0.1.1-dev.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE _timescaledb_catalog.hypertable ADD UNIQUE (id, schema_name);
+ALTER TABLE _timescaledb_catalog.hypertable_index DROP CONSTRAINT hypertable_index_hypertable_id_fkey;
+ALTER TABLE _timescaledb_catalog.hypertable_index ADD CONSTRAINT hypertable_index_hypertable_id_fkey FOREIGN KEY (hypertable_id, main_schema_name) REFERENCES _timescaledb_catalog.hypertable(id, schema_name) ON UPDATE CASCADE ON DELETE CASCADE;
+
+ALTER TABLE _timescaledb_catalog.chunk_index
+DROP CONSTRAINT chunk_index_main_schema_name_fkey,
+ADD CONSTRAINT chunk_index_main_schema_name_fkey
+FOREIGN KEY (main_schema_name, main_index_name) 
+REFERENCES _timescaledb_catalog.hypertable_index(main_schema_name, main_index_name) 
+ON UPDATE CASCADE 
+ON DELETE CASCADE;
+
+
+CREATE TABLE IF NOT EXISTS _timescaledb_catalog.hypertable_trigger (
```

Comment:
I wonder if IF NOT EXISTS might be dangerous here as we probably shouldn't run an update if it does exist.